addons/project/models/project.py

```diff
@@ -699,6 +700,28 @@ def unlink(self):
         analytic_accounts_to_delete.unlink()
         return result
 
+    @api.model
+    def _search(self, args, offset=0, limit=None, order=None, count=False, access_rights_uid=None):
+        order_items = [order_item.strip().lower() for order_item in (order or self._order).split(',')]
+        new_order, item_index, desc = [], -1, False
+        for index, item in enumerate(order_items):
+            if 'is_favorite' in item:
+                item_index = index
+                if 'desc' in item:
```

Comment:
I think we could also have `DESC` and so it could not work in that case. 😢 
I think we could do something like that:
```suggestion
                if item.lower().endswith('desc'):
```